<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameWindow</class>
 <widget class="QWidget" name="GameWindow">
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="score_label">
     <property name="text">
      <string>Score: 0 | High Score: 0</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="instructions_label">
     <property name="text">
      <string>Press Space to Jump</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="start_btn">
     <property name="text">
      <string>Start Game</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
